apps/condo/pages/ticket/index.tsx

```diff
@@ -361,10 +372,45 @@ const TicketsTableContainer = ({
         skip: (currentPageIndex - 1) * DEFAULT_PAGE_SIZE,
     })
 
+    const [loadNewTicketCount] = useLazyQuery(TicketGQL.GET_COUNT_OBJS_QUERY, {
+        onCompleted: ({ meta: { count } }) => {
+            if (!isNull(ticketsCountRef.current) && ticketsCountRef.current !== count) {
+                const totalNewTicketsCount = count - ticketsCountRef.current
```

Comment:
What if some tickets will be deleted?
```suggestion
            if (!isNull(ticketsCountRef.current) && ticketsCountRef.current !== count) {
                const totalNewTicketsCount = Math.max(count - ticketsCountRef.current, 0)
                if (totalNewTicketsCount > 0) { ...
```

or

```suggestion
            if (!isNull(ticketsCountRef.current) && ticketsCountRef.current < count) {
                const totalNewTicketsCount = count - ticketsCountRef.current
```

Reply:
Good suggestion, fixed it)